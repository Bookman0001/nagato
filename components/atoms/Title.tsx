import React from 'react'
import styled from 'styled-components'

interface Props {
  children: React.ReactNode
}

export default function Title({ children }: Props) {
  return (
    <TitleContainer>
      <>{children}</>
    </TitleContainer>
  )
}

const TitleContainer = styled.h2`
  font-size: 32px;
  span {
    font-size: 64px;
  }
`
