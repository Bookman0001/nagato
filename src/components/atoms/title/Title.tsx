import React from 'react'
import styled from 'styled-components'

import { deviceWidth, fontSize } from 'src/theme/constants'

interface Props {
  children: React.ReactNode
}

export function Title({ children }: Props) {
  return <TitleContainer>{children}</TitleContainer>
}

const TitleContainer = styled.h2`
  display: flex;
  justify-content: center;
  align-items: center;
  line-height: 1.5;
  padding-bottom: 2rem;
  font-size: ${fontSize.L};
  @media (max-width: ${deviceWidth.PHONE}) {
    font-size: ${fontSize.M};
  }
`
