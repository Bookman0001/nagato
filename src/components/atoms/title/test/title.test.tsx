import { shallow } from 'enzyme'

import Title from '../'

test('Title', () => {
  const component = shallow(<Title>test</Title>)
  expect(component).toMatchSnapshot()
})
