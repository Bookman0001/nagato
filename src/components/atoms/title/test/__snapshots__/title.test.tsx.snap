// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title 1`] = `
<styled.h2>
  test
</styled.h2>
`;
